docs/docs/understand/projects.md

```diff
@@ -5,8 +5,24 @@ sidebar_position: 20
 
 # Projects in Minder
 
-Projects in Minder are a way to group entities together. They are a way to organize your entities (repositories, artifacts,
-etc.) based on the policy you want to enforce or the team that owns them.
+Projects in Minder are a way to group entities together, and to share access
+with other people in your organization. They are a way to organize your entities
+(repositories, artifacts, etc.) based on the policy you want to enforce or the
+team that owns them.
 
-When creating an account, Minder will automatically create a default project for you. You can create additional projects as
-a way to organize and secure your entities, and manage access for your team members.
+Each user in an project is [assigned a role](../user_management/user_roles.md),
+such as `editor` or `admin`. Projects must have at least one administrator, who
+has permission to
+[invite other users to the project](../user_management/adding_users.md) and
+change users roles within the project.
+
+When creating an account, Minder will automatically create a default project for
+you, unless you are accepting an invitation to an existing project. You can
```

Comment:
Unfortunately this is not like that right now. We still create a project, we just don't go through the onboarding flow and also show by default the oldest project in the UI (which is the one the user accepted the invite for). 

We had a discussion with @jhrozek that we should separate the project creation part from the user creation part, ref. #3826 (implying changes on the client's side - UI and CLI)

Reply:
@ethomson -- are we okay shipping invitations with this change from the PRD & tech design, or do we need to address it before release?

Reply:
Another note - if the user accepted an invite, but then selects their default project from the drop down menu they will be taken through the onboarding flow for it since it's what the front end does for every new project.

Reply:
* I think it's really surprising that we new up a secret project behind the scenes that nobody asked for.
* We have a metric around project creation and how many have rules and profiles associated with them (versus not), and this will annihilate that metric. 🙃 
* The UI should show the most recent project, not the oldest project. (Again, the FE needs a data store for its own business logic, and doesn't have one.)

Can you give me a rough cost on separating user creation from project creation?

Reply:
I _think_ this will be a small change, if I'm right, I'm about 2/3rds done now (plus tests, so really 40% done).

Reply:
I made it a medium-size change, and it will need a client release: https://github.com/stacklok/minder/pull/3909

Reply:
Also @kantord for a heads-up; I commented on #3909, but the frontend should not call `CreateUser` before calling `ResolveInvitation` -- `ResolveInvitation` will take care of any bookkeeping needed in terms of getting user information.  Note that before calling `ResolveInvitation`, the `GetUser` call will also fail.